### PR TITLE
[4.0] Show Password

### PR DESCRIPTION
**Pull request by maliouris**
Pull Request for Issue # .
#24312 
### Summary of Changes

Make eye icon on login forms accessible. Change it from span to input. Addittionally sr-only text now changes if password is seen or not.

### Testing Instructions

Go to a login form, for example admin login form and try to select icon eye with keyboard

### Expected result

Navigate through the forms with tabs

### Actual result

Skip eye icon as it is span

### CODE REVIEW

**Comment by brianteeman** [comment]
Please edit your post to indicate exactly what the problem is, what you have changed and how to test that this has fixed the problem.

**Comment by maliouris** [comment]
Is it ok now ?

File: administrator/modules/mod_login/tmpl/default.php [hunk 1]
```diff
@@ -51,9 +51,9 @@ class="form-control input-full"
 				<?php echo Text::_('JGLOBAL_PASSWORD'); ?>
 			</label>
 			<div class="input-group">
-				<span class="input-group-prepend">
+				<span class="input-group-prepend input-group-text">
 					<span class="sr-only"><?php echo Text::_('JSHOW'); ?></span>
-					<span class="input-group-text icon-eye" aria-hidden="true"></span>
+                    <input type="checkbox" class="icon-eye" aria-hidden="true">
```
Comment (Quy) [inline administrator/modules/mod_login/tmpl/default.php]: Use tabs and not spaces to indent.

File: templates/cassiopeia/scss/template.scss [hunk 2]
```diff
@@ -64,3 +64,19 @@
   width: 100% !important;
   max-width: 240px;
 }
+.input-group-prepend {
+  margin-right: 0px; }
```
Comment (Quy) [inline templates/cassiopeia/scss/template.scss]: Move `}` to its own line.

**Comment by dgrammatiko** [comment]
FWIW before doing any accessibility improvements first the field **needs** to be converted to custom element, eg https://github.com/joomla/joomla-cms/pull/23215

**Comment by maliouris** [comment]
As far as I saw your pull request didnt make it to the main branch because it was a custom element. Am I wrong? If I have to make a custom element I can close this pull request and prepare a new one.

**Comment by HRIT-Florian** [comment]
I have tested this item :red_circle: unsuccessfully on 23977f1719c7622d5e672644af267cc387aba57e<br /><br />Meanwhile we have a button after the span in Joomla, so it is solved already. 
So it is accessible via keyboard and the PR has a conflict, missing bootstrap.scss
PR should be closed ;)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24326">issues.joomla.org/tracker/joomla-cms/24326</a>.</sub>

**Comment by HLeithner** [comment]
> I have tested this item  unsuccessfully on [23977f1](https://github.com/joomla/joomla-cms/commit/23977f1719c7622d5e672644af267cc387aba57e)
> 
> Meanwhile we have a button after the span in Joomla, so it is solved already.
> So it is accessible via keyboard and the PR has a conflict, missing bootstrap.scss
> PR should be closed ;)
> 
> This comment was created with the [J!Tracker Application](https://github.com/joomla/jissues) at [issues.joomla.org/tracker/joomla-cms/24326](https://issues.joomla.org/tracker/joomla-cms/24326).

Thats partly true, the mod_login is not support this feature. if we don't get a response from @maliouris within 2 week we should close this pr and create a new one.

**Comment by Quy** [comment]
Closing due to no response.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24326">issues.joomla.org/tracker/joomla-cms/24326</a>.</sub>

**Pull request closed without merging**